dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/trackedentity/AbstractTrackedEntityInstanceService.java

```diff
@@ -176,6 +180,14 @@
     // READ
     // -------------------------------------------------------------------------
 
+    private Set<TrackedEntityAttribute> mergeIf( Set<TrackedEntityAttribute> set1, Set<TrackedEntityAttribute> set2, boolean condition )
```

Comment:
I feel like "condition" is a bit confusing here. Could you write a comment about the intention? If its true we merge, and if its not true we dont?